### PR TITLE
Feat: Update scrollbar styles in custom.css

### PR DESCRIPTION
**Pull request by arpy8**
## Related Issue

Fix: #734 

## Description

Added proper styling for the scrollbar to match website's color theme.

## Type of PR

- [x] Feature enhancement

## Screenshots / Videos (if applicable)

| before                                                                                                                  | after                                                                                                                   |
|-------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------|
| image | image |

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 2px grey;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #47a94a;
```
Comment (Ajay-Dhangar) [inline src/css/custom.css:13]: replace `background: #47a94a;` to `background: var(--ifm-color-primary)`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by Ajay-Dhangar** [comment]
@arpy8 could you please update it? Otherwise, I may need to close it.

**Comment by arpy8** [comment]
Yes, I'll update it asap

**Review by Ajay-Dhangar — APPROVED** [review]
done

**Pull request merged**